staging/src/k8s.io/apiserver/pkg/storage/value/transformer.go

```diff
@@ -32,6 +32,15 @@ type Context interface {
 	AuthenticatedData() []byte
 }
 
+// KMSService allows encrypting and decrypting secrets using an external service, which provides a
+// Key-Encryption-Key (KEK). This is needed to implement envelope encryption based transformers.
```

Comment:
I think you can take out the last sentence here. That's an implementation detail.